site-new/README.md

```diff
@@ -0,0 +1,43 @@
+# Armeria website
+
+> WARNING: This new project is now under construction, and will replace the current Gatsby project.
+
+This directory contains the source code for the official Armeria website, built with Docusaurus.
+
+### Installation
+
+```
+$ yarn
```

Comment:
Question) Should we use `npm` for consistency? We use `npm` to install dependencies and run scripts for other JS modules.

Reply:
Changed it to use npm.
I was unsure about which package manager to use, but seeing questions arise, I think it's better to maintain consistency.